Javascript/q40.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>q40</title>
+
+    <style>
+      body {
+        font-family: Verdana, Geneva, Tahoma, sans-serif;
+        font-size: small;
+      }
+
+      .flex {
+        display: flex;
+        gap: 4px;
+        flex-wrap: wrap;
+        width: 240px;
+      }
+    </style>
+
+    <script>
+      function onClicking() {
```

Comment:
The name is almost good. `onClicking` suggests that this function somehow gets called when the user keeps on clicking it. Which makes no sense. A better name is `onClick` because it suggests that this function get called when the button gets clicked and thats it.

Reply:
ok